src/lib.rs

```diff
@@ -6,70 +6,204 @@
 //! Turn Traditional Chinese script to Simplified Chinese script and vice-versa. Check string script to determine if string is Traditional or Simplified Chinese.
 //!
 //! ### Usage
-//! ```rust
+//! ```
 //! extern crate character_converter;
 //!
-//! use character_converter::CharacterConverter;
+//! use character_converter::*;
 //!
-//! let converter: CharacterConverter = CharacterConverter::new();
 //!
-//! let traditional_text = "欧洲";
-//! let simplified_text = "歐洲";
+//! let traditional_text = "歐洲";
+//! let simplified_text = "欧洲";
 //!
 //! // Check script
-//! let result_one: bool = converter.is_traditional(traditional_text);
-//! println!("{}", result_one); // --> true
+//! assert!(is_traditional(traditional_text));
 //!
-//! let result_two: bool = converter.is_simplified(traditional_text);
-//! println!("{}", result_two); // --> false
+//! assert!(!is_simplified(traditional_text));
 //!
 //! // Convert script
-//! let result_three: String = converter.traditional_to_simplified(traditional_text);
-//! println!("{}", result_three == simplified_text); // --> true
+//! let result_three = traditional_to_simplified(traditional_text);
+//! assert!(result_three == simplified_text);
 //!
-//! let result_four: String = converter.simplified_to_traditional(simplified_text);
-//! println!("{}", result_four == traditional_text); // --> true
+//! let result_four = simplified_to_traditional(simplified_text);
+//! assert!(result_four == traditional_text);
+//! ```
+#![cfg_attr(feature = "bench", feature(test))]
 
 extern crate bincode;
 
-mod character_converter;
-pub use self::character_converter::Converter as CharacterConverter;
+use std::collections::HashMap;
+
+use bincode::deserialize_from;
```

Comment:
What I don't quite like is that we use `serde` and `bincode` as the encoding format where we could have simply used a CSV file.

Reply:
The serde bincode encoder doesn't do much more than putting the strings one after the other and doesn't compress the content or anything.

Reply:
I don't plan to change this in this PR